src/components/auth/LoginForm.scss

```diff
@@ -12,13 +12,31 @@
     color: lightslategray;
   }
 
-  button {
-    background-color: green;
-    margin-top: 1.5em;
-    font-weight: bold;
-    margin-left: auto;
-    margin-right: auto;
-    padding: 1rem 2rem;
-    border-radius: .3rem;
+  &__buttons {
+    display: flex;
+    margin-top: 16px;
+
+    button {
```

Comment:
Not really your fault here but we should avoid this and use custom button components like `SubmitButton` with a proper styling for these cases.